### PR TITLE
How to pitch the Foundation for Public Code

### PR DESCRIPTION
**Pull request by ElenaFdR**
Still to do on this draft PR:

- [x] check reasonable hyperlinks are added (including to FSFE)
- [x] add transcription of @bvhme's filmed pitches (and link to films?)
- [x] standardization on bullets - some start with caps and have fullstops at the end, some don't

@nebster, do you have any easy pitches you use when you start talking to a stranger?

@everyone, is the tone of this helpful? Can I delete the 'audience' H2 since the actual pitches have different audience divisions?

-----
[View rendered activities/communication/pitching.md](https://github.com/publiccodenet/about/blob/batter-up/activities/communication/pitching.md)

### CODE REVIEW

**Comment by Ainali** [comment]
Yes, to me the tone is definitely helpful. And yes, the "audience" H" can be removed, it isn't really adding value know.

File: activities/communication/pitching.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+These are pitches we already use.
+
+For developers, tech people and open source enthusiasts:
+
+* "You know the Free Software Foundation Europe's (FSFE) [Public Money, Public Code] campaign? We're helping public organizations actually do it."
+
+For people with government experience or interest:
+
+* "We're helping public organizations that develop their own software to collaborate and share it with other public organizations so that the public doesn't need to duplicate the work."
+* [Boris pitches]
```
Comment (AlbaRoza) [inline activities/communication/pitching.md]: * **Digital transformation**. "Every government is going through digital transformation but they are way more governments in the world that are going through digital transformations than there are technologists that understand that or programmers that can help with that. We need to be able to work together, it's either we work together or just we give keys to companies that'll sell us things, and then they'll keep us in their grid. So we need to learn how to work together and as the Foundation we are trying to build that collaboration at scale."
* **Mutual self help institution**. "We help public organizations to work together in order to go through digital transformation in a way they control and they are able to deal with. However, it's in none of their mandates to help others, yet real collaboration doesn't ever happen if you don't help each other and I think the Foundation for Public Code adds a player to that field where everyone can help each other by helping themselves."
* **Open source for public organizations**. "We are an open source foundation like you have the Eclipse Foundation or the Linux Foundation but then for public organizations. Public organizations are quite different so they need a different approach, but they can still benefit from that type of collaboration or trying to figure out how that works."
* **Software collaboration is better**. "We envision a future where software, as part of government, is taken a seriously as policy is now, dus being open becomes really crucial. That comes with the added advantage that you can then be critical with each others' open software and you can work together to make everyone's software better."

Comment (ElenaFdR) [reply activities/communication/pitching.md]: Done! Thank you!

**Comment by ElenaFdR** [comment]
Team, VSCode's Markdown delinter isn't spotting the same issues as Travis, nor can my human eyes see them. Can someone with another delinter have a look?

**Comment by ElenaFdR** [comment]
@demkodo Great spot on the typos! I've fixed those now.

**Comment by ElenaFdR** [comment]
Hey, if this is approved, I'm happy for anyone to merge this before I get back.

**Comment by bvhme** [comment]
To me it seems important that we find a way to interlink this and our _value proposition mapping_ sections, as the risk of duplicating content is high and thus also the risk of creating content debt.

**Comment by ElenaFdR** [comment]
@bvhme I've crosslinked the pitches and the value propositions in a way that explains the difference. I haven't removed the audience list from the pitching game proposal, in part because it was more naunced than our value proposition list.

**Pull request merged**